src/Jackett.Common/Indexers/PolishTracker.cs

```diff
@@ -16,21 +18,17 @@ namespace Jackett.Common.Indexers
     public class PolishTracker : BaseWebIndexer
     {
         private string LoginUrl => SiteLink + "login";
-        private string TorrentApiUrl => SiteLink + "apitorrents";
-        private string CDNUrl => "https://cdn.pte.nu/";
+        private string SearchUrl => SiteLink + "apitorrents";
+        private static string CdnUrl => "https://cdn.pte.nu/";
 
-        public override string[] LegacySiteLinks { get; protected set; } = new string[] {
+        public override string[] LegacySiteLinks { get; protected set; } = {
             "https://polishtracker.net/",
-            };
+        };
 
-        private new ConfigurationDataBasicLoginWithEmail configData
-        {
-            get => (ConfigurationDataBasicLoginWithEmail)base.configData;
-            set => base.configData = value;
-        }
+        private new ConfigurationDataBasicLoginWithEmail configData => (ConfigurationDataBasicLoginWithEmail)base.configData;
```

Comment:
Does this work when changing the config values (delete and re-add)? I've been leaving set in for that purpose.

Reply:
Yest, it works.

Reply:
huh. Cool.